test/integration/peggy-e2e-test.md

```diff
@@ -12,46 +12,63 @@ sifnoded start
-5. check the ballance of contract before lock
-yarn peggy:getTokenBalance  0x2C2B9C9a4a25e24B174f26114e8926a9f2128FE4  eth
-6. check the user2 balance before lock
-sifnodecli query account $(sifnodecli keys show user2 -a)
+4. check the balance of operator after lock
+yarn peggy:getTokenBalance 0xf17f52151EbEF6C7334FAD080c5704D77216b732 \ # any ethereum account
+eth
+5. check the ballance of contract after lock
+yarn peggy:getTokenBalance 0x75c35C980C0d37ef46DF04d31A140b65503c0eEd \ # BridgeBank contract address
+eth
+6. check the cosmos user balance after lock
+sifnodecli query account $(sifnodecli keys show akasha -a)
 
-### case 2: burn user2's eth in cosmos then asset to back to ethereum's validator account
-1. check the validator's balance before burn
-yarn peggy:getTokenBalance 0xf17f52151EbEF6C7334FAD080c5704D77216b732 eth
-sifnodecli query account $(sifnodecli keys show user2 -a)
+### case 2: burn user's ceth in cosmos then asset to back to ethereum's receiver account
+1. check the receiver's balance before burn
+yarn peggy:getTokenBalance 0xf17f52151EbEF6C7334FAD080c5704D77216b732 \ # any ethereum receiver account
+eth
+sifnodecli query account $(sifnodecli keys show akasha -a)
 
 2. send burn tx in cosmos
-sifnodecli tx ethbridge burn $(sifnodecli keys show user2 -a) 0xf17f52151EbEF6C7334FAD080c5704D77216b732 1000000000000000000 peggyeth --ethereum-chain-id=5777 --from=user2 --yes
+sifnodecli tx ethbridge burn $(sifnodecli keys show akasha -a) \ # cosmos account from
+0xf17f52151EbEF6C7334FAD080c5704D77216b732 \ # ethereum receiver account
+100 ceth --ethereum-chain-id=5777 \ # web3.eth.getChainId()
+--from=akasha --yes
 
-3. check user2's account 
-yarn peggy:getTokenBalance 0xf17f52151EbEF6C7334FAD080c5704D77216b732 eth
-sifnodecli query account $(sifnodecli keys show user2 -a)
+3. check both accounts
+yarn peggy:getTokenBalance 0x627306090abaB3A6e1400e9345bC60c78a8BEf57 eth
+sifnodecli query account $(sifnodecli keys show akasha -a)
 
 ### case 3: lock rowan in cosmos then issue the token in ethereum
-sifnodecli tx ethbridge lock $(sifnodecli keys show user2 -a) 0xf17f52151EbEF6C7334FAD080c5704D77216b732 1 rwn  --ethereum-chain-id=5777 --from=user2 --yes
+1. sifnodecli tx ethbridge lock $(sifnodecli keys show akasha -a) 0xf17f52151EbEF6C7334FAD080c5704D77216b732 \ # receiver account in ethereum network
+1 rwn --ethereum-chain-id=5777 --from=akasha --yes
+
+2. get newly created "eRWN" token address
+yarn peggy:getTokenAddress eRWN
 
-1. check the balance of user2 peggyatom in ethereum
-yarn peggy:getTokenBalance 0xf17f52151EbEF6C7334FAD080c5704D77216b732  0x409Ba3dd291bb5D48D5B4404F5EFa207441F6CbA
-sifnodecli query account $(sifnodecli keys show user2 -a)
+3. check both balances rwn and eRWN
+yarn peggy:getTokenBalance 0xf17f52151EbEF6C7334FAD080c5704D77216b732 eRWN
+sifnodecli query account $(sifnodecli keys show akasha -a)
 
 ### case 4: burn rowan in ethereum and rowan will be back to cosmos
```

Comment:
"burn rowan in ethereum and rowan will be back to cosmos" - add the symbols?  

"burn erowan (eRWN) in ethereum and rowan (rowan) will be back to cosmos"

(I'm not sure these are the right symbols)